templates/README.md

```diff
@@ -28,53 +26,84 @@ The `config.yaml` file has two keys:
 - `global` with a map of values available for all templates
 - `templates` with a list of files to generate
 
-The `config.yaml` serves as the input file for the Render Templates that generates the jobs based on the file definition and templates which it specifies. The `config.yaml` file defines the names of the output file, their location, and configuration referred to in `values`.
+The .yaml files in `data` directory have one key:
 
-See the example of `console-backend-service` in which the `generic-component.yaml` template is used to create the component and test-related YAML files using values defined by the **kyma_generic_component** parameter.
+- `templates` with a list of files to generate
 
-```yaml
-kyma_generic_component: &kyma_generic_component
-  repository: github.com/kyma-project/kyma
-  pushRepository: kyma
-  bootstrapTag: v20181204-a6e79be
-  additionalRunIfChanged:
-    - ^scripts/
+The `config.yaml` and .yaml files in the `data` directory serves as the input file for the Render Templates that generates the jobs based on the file definition and templates which it specifies. These files define the names of the output file, their location, and configuration referred to in `values`.
```

Comment:
```suggestion
The `config.yaml` and .yaml files in the `data` directory serve as the input files for the Render Templates, which generate the jobs based on the file definition and templates that it specifies. These files define the names of the output file, their location, and configuration referred to in `values`.
```

"...that it specifies" - what is "it" ? the directory? Can we maybe split the first sentence in two?

Reply:
"It" should be the files, each one contains information about used template and all values used during the generation.